msteams-platform/apps-in-teams-meetings/teams-together-mode.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Together Mode in Teams
+description: Work with Together Mode 
+ms.topic: conceptual
+---
+
+# Together Mode in Teams
+
+Teams Together Mode provides a simple, innovative, and user-friendly meeting environment. It digitally combines participants into a single virtual space instead of being shown separately in boxes. It creates your virtual image and places the image in a virtual environment where you can easily interact with your peers. It makes meetings more engaging in the auditorium view.
+
+> [!VIDEO https://www.youtube-nocookie.com/embed/MGsNmYKgeTA]
+
+You can create scene only apps or extend your current app to have scenes for Together Mode. A scene is a virtual view of the auditorium or meeting venue where the attendees are placed and includes the seats for attendees.
+
+>[!NOTE]
+> Scene only apps are recommended as the acquisition experience for such apps is more seamless.
+
+The following image gives an overview of adding a scene to an app:
+
+![Add scene to app](../assets/images/apps-in-meetings/scene-added-to-app.png)
+
+>[!NOTE]
+> Multiple scenes in a single app package appear as a flat list of scenes to users.
+
+There are prerequisites that you must complete before using the Together Mode.
+
+## Prerequisites
+
+You must have a basic understanding of the following to use Together Mode:
```

Comment:
I think the pre-reqs here are very vague. Please replace with this following: 

1. Definition of Scene and Seats in a scene 
2. Have a Microsoft Developer account and familiarity with the Microsoft Teams Dev Center and App Studio
3. Concept of app sideloading (insert a link here to this page : https://review.docs.microsoft.com/en-us/microsoftteams/platform/concepts/deploy-and-publish/apps-upload?branch=pr-en-us-2710